data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg

```diff
@@ -312,12 +312,13 @@
 
         [message]
             speaker=Deoran
-            message= _ "Look! The undead army has broken through Sir Gerrick’s lines! Alas, he is lost!"
+            image=portraits/deoran-sad.png
+            message= _ "Look there! The undead army has broken through Sir Gerrick’s line! Alas, he is lost!"
         [/message]
 
         [message]
             speaker=Moreth
-            message= _ "Since I was a boy, Gerrick was my leader, and there was no one braver. Today we shall defend our homes, and avenge him!"
+            message= _ "Gerrick was my mentor and caretaker ever since I was little. I wish he had not sacrificed himself for us, but his bravery and courage is truly admirable. Today, we will defeat the undead and avenge him!"
```

Comment:
Is it possible for Moreth to not be present? If so, his line is omitted. This might make the rest ("I, too, wish") seem odd, There might be a way to replace Moreth, if need be, with another human recruited (from Westin, presumably). Otherwise, your changes, below, will need more work.